R/estimator-helpers.R

```diff
@@ -87,7 +87,7 @@ get_weights <- function(data, estimator) {
 #' @param estimator Either `NULL` for auto-selection, or a single character
 #' for the type of estimator to use.
 #'
-#' @seealso [class_metric_summarizer()] [metric_vec_template()]
+#' @seealso [metric-summarizers()] [metric-vec_template()]
```

Comment:
Do these links really work with the `-` and the `()`?

Reply:
They match the `@name` for the summarizer functions. I opted to use the `-` instead of `_` because it becomes very close to the deprecated `metric_summarizer()`

![Kapture 2022-11-22 at 15 49 55](https://user-images.githubusercontent.com/14034784/203444049-575ff6ac-d2ef-4b03-ada0-96dc6ba6373b.gif)